src/components/FooterContainer/index.tsx

```diff
@@ -65,37 +18,7 @@ const StyledFooterBottom = styled(FooterBottom)`
 
 const FooterContainer = () => (
   <Footer>
-    <FooterWrapper className="no-print" data-testid="siteFooter">
-      <Container>
-        <Disclaimer data-testid="disclaimer">
-          <StyledHeading forwardedAs="h2">
-            Lukt het niet om een melding te doen?
-          </StyledHeading>
-          <span
-            // eslint-disable-next-line react/no-danger
-            dangerouslySetInnerHTML={{
-              __html: stringFormatter(configuration.language.footer1, {
-                '##GENERAL_PHONE_NUMBER_FORMATTED##':
-                  configuration.language.phoneNumber.replace(' ', ''),
-                '##GENERAL_PHONE_NUMBER##': configuration.language.phoneNumber,
-              }),
-            }}
-          />
-          <br />
-          <span>{configuration.language.footer2}</span>
-        </Disclaimer>
-
-        <span
-          // eslint-disable-next-line react/no-danger
-          dangerouslySetInnerHTML={{
-            __html: stringFormatter(configuration.language.avgDisclaimer, {
```

Comment:
`avgDisclaimer` is removed as well, intentionally? Has it been placed somewhere else?

Reply:
there is a privacy link in the footer bottom that links to a new page on avg

Reply:
And phonenumber is still in used, in the traffic form, for instance

Reply:
Where is that privacy link then? I don't see it in this diff.

Reply:
Oh, it was already there all the time :)